Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -92,6 +92,29 @@ public override void Draw(RectangleF visibleRectangle)
             _spriteBatch.End();
         }
 
```

Comment:
Read the description, this is just a quick fix for me, plus I didn't bother setting up the rendering code since what I'm working on is using 1 map per screen that's always fully visible.